dist/leaflet.distortableimage.js

```diff
@@ -1133,14 +1133,27 @@ L.Map.addInitHook('addHandler', 'boxSelector', L.Map.BoxSelectHandle);
 L.DomUtil = L.DomUtil || {};
 L.DistortableImage = L.DistortableImage || {};
 
-L.DistortableImage.Keymapper = L.Control.extend({
-    initialize: function(options) {
-        L.Control.prototype.initialize.call(this, options);
+L.DistortableImage.Keymapper = L.Handler.extend({

```

Comment:
So here, yeah - sorry, we don't need this to extend `Handler` - that was just a fragment of the code i was quoting. I mainly meant just putting this all in `L.DistortableImage.Keymapper` - which looks great!